### PR TITLE
3179: Use explicit list of unofficial country names

### PR DESCRIPTION
**Pull request by edmofro**
### Issue #:

https://github.com/beyondessential/tupaia-backlog/issues/3179

### CODE REVIEW

File: packages/utils/src/getCountryCode.js [hunk 1]
```diff
@@ -4,19 +4,20 @@
...
+};
 
+export const getCountryCode = countryName => {
   const { getCode: getCountryIsoCode } = require('countrynames');
-  // Use the country ISO code if there's a direct match,
-  //otherwise base on the two letter facility code prefix
   const countryCode =
-    getCountryIsoCode(countryName) || entityObjects.length
-      ? entityObjects[0].code.substring(0, 2)
-      : null;
```
Comment (edmofro) [inline packages/utils/src/getCountryCode.js:18]: The problem here was that I'm guessing this code intended to act as a lazily evaluated conditional with an early return (i.e., if getCountryIsoCode returned something, use that), but instead the whole || was just used as the condition for the ternary operator. That results in "if there's a valid ISO code for this country name, use the first two letters of the first entry in the spreadsheet", which is not what anyone would have wanted, I'm sure.

File: packages/meditrak-server/src/routes/importEntities/assertCanImportEntities.js [hunk 2]
```diff
@@ -6,9 +6,9 @@ import { getCountryCode } from '@tupaia/utils';
 import { TUPAIA_ADMIN_PANEL_PERMISSION_GROUP } from '../../permissions';
 
 export const assertCanImportEntities = async (accessPolicy, entitiesByCountryName) => {
```
Comment (kael89) [inline packages/meditrak-server/src/routes/importEntities/assertCanImportEntities.js:8]: It seems this function requires just country names now:

```suggestion
export const assertCanImportEntities = async (accessPolicy, countryNames) => {
```

File: packages/utils/src/getCountryCode.js (hunk 1 above)
Comment (kael89) [reply packages/utils/src/getCountryCode.js:18]: Great explanation!

**Review by kael89 — APPROVED** [review]
Looks good! I left one suggestion but it's minor, approving

File: packages/meditrak-server/src/routes/importEntities/assertCanImportEntities.js (hunk 2 above)
Comment (edmofro) [reply packages/meditrak-server/src/routes/importEntities/assertCanImportEntities.js:8]: Thorough review for a quick pR! I've made that change, good suggestion

**Pull request merged**